backend/app/views/spree/admin/promotions/index.html.erb

```diff
@@ -82,7 +82,15 @@
           </td>
           <td>
             <span class="pill pill-<%= promotion.active? ? 'active' : 'inactive' %>">
-              <%= t(promotion.active?? :active : :inactive, scope: 'spree') %>
+              <% if promotion.active? %>
+                <%= t(:active, scope: 'spree') %>
```

Comment:
what about adding a scope to this translation? Like `spree.promotion_status.active`?